lib/synced/strategies/updated_since.rb

```diff
@@ -44,7 +44,11 @@ def initial_sync_since
 
       def updated_since
         instrument("updated_since.synced") do
-          [@timestamp_strategy.last_synced_at, initial_sync_since].compact.max
+          last_synced_at_offset = ENV.fetch("LAST_SYNCED_AT_OFFSET", 0).to_i
```

Comment:
I'd extract this to `last_synced_at` private method where we could advance with `ENV.fetch("SYNCED_TOLERANCE", 600)`. 10 minutes sounds like a fair default tolerance for me.